### PR TITLE
Adds some emotes

### PR DESCRIPTION
**Pull request by pixelkitty286**
## About The Pull Request

Adds some emotes

## Why It's Good For The Game

MRRP MRP MEOW! You play like a cat
As much as I loath Tesh I like Prbt

## Changelog

:cl:
add: emotes: mrrp, fpurr, gecker, and prbt
sound: mrrp.ogg, prbt.ogg, fox_purr.ogg, and foxgecker.ogg
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Emote code needs to go in the zubbers folder, not the skyrat folder

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
Yup. What moth said.

**Comment by pixelkitty286** [comment]
OH LOL Hold on let me fix that

File: modular_zubbers/modules/emotes/code/emotes.dm [hunk 1]
```diff
@@ -23,3 +23,35 @@
...
+	key_third_person = "prbts!"
+	message = "prbts!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'modular_zubbers/modules/emotes/sound/voice/prbt.ogg'
+
+/datum/emote/living/gecker
+	key = "gecker"
+	key_third_person = "geckers"
+	message = "geckers."
```
Comment (theselfish) [inline modular_zubbers/modules/emotes/code/emotes.dm:54]:
```suggestion
	message = "geckers!"
```
Should make this an exclamation for consistency.

**Comment by pixelkitty286** [comment]
I love CI Suite failing with an error that is not my doing.

**Comment by StrangeWeirdKitten** [comment]
> I love CI Suite failing with an error that is not my doing.

Sometimes it happens.

**Review by StrangeWeirdKitten — APPROVED** [review]

**Comment by StrangeWeirdKitten** [comment]
@pixelkitty286 before I merge this, I need to know where you got these sounds.

**Comment by pixelkitty286** [comment]
> @pixelkitty286 before I merge this, I need to know where you got these sounds.
mrrp.ogg or cat_meow_after_jump_alt06.ogg is from the videogame Stray from B12 studios.
prbt.ogg unknown file is from CitRP under the misc sound folder. Where they got it from is unknown.
fox_purr.ogg & foxgecker.ogg from discord user izzyyess. Where they got it from is unknown.

**Pull request merged**